app/views/playbook/principles.html.slim

```diff
@@ -0,0 +1,264 @@
+doctype html
+html lang="#{I18n.locale}"
+
+  = render 'head'
+
+  body
+    .site-wrap
+      = render 'shared/usa_banner'
+
+      nav.bg-white
+        .container.cntnr-wide.px2.py3
+          = render 'nav_top'
+
+              = link_to(playbook_principles_path(anchor: 'pb-flexible'),
+                class: 'py3 pl3 pr5 bg-white block pb-principle-cntnr') do
+
+      .pb-bg-navy
+        .container.cntnr-wide.px2.py5
+          h2.mt0.mb1.h1.white The principles
+          = image_tag(asset_url('playbook/hr-red-1.svg'), alt: 'hr', class: 'mb3')
+          .clearfix.mxn2.pb-lead
+            .sm-col.sm-col-8.px2.white
+              .serif
+                p
+                  | Building this system includes taking recommendations and best practices from
+                    policy experts, system architects, and people focused on creating a great user
+                    experience. Our research and our progress so far has led us to identify five
+                    major principles that we believe will help us make the best system possible.
+                p
+                  | If you are adopting the system we’re building or creating your own, please use
+                    these principles as a guide to ensure a reliable product that deserves the
+                    public trust.
+              ul.mt3.mb0.ml1.pl1.pb-list--bullet
+                li.mb2
+                  = link_to('Focus on user needs',
+                    playbook_principles_path(anchor: 'pb-users'), class: 'white bold')
+                li.mb2
+                  = link_to('Be transparent about how it works',
+                    playbook_principles_path(anchor: 'pb-transparent'), class: 'white bold')
+                li.mb2
+                  = link_to('Build a flexible product',
+                    playbook_principles_path(anchor: 'pb-flexible'), class: 'white bold')
+                li.mb2
+                  = link_to('Use modern privacy practices',
+                    playbook_principles_path(anchor: 'pb-privacy'), class: 'white bold')
+                li.mb2
+                  = link_to('Create responsive security systems',
+                    playbook_principles_path(anchor: 'pb-security'), class: 'white bold')
+            .sm-col.sm-col-4.px2.center.sm-show
+              = image_tag(asset_url('playbook/graphic-playbook.png'),
+                alt: APP_NAME, width: 244, class: 'mt2')
+
+      .bg-white
+        .container.cntnr-wide.px2.py5
+          .clearfix
+            .sm-col.sm-col-8
```

Comment:
i was keeping the right area open for another column for the right nav, but since it will probably be positioned absolutely i think you're right that its unnecessary.